module-os/CMakeLists.txt

```diff
@@ -67,23 +54,19 @@ target_include_directories(${PROJECT_NAME}
 
         ${CMAKE_CURRENT_SOURCE_DIR}
 
-        ${CMAKE_CURRENT_SOURCE_DIR}/FreeRTOS/include
-        ${CMAKE_CURRENT_SOURCE_DIR}/FreeRTOS
-        ${CMAKE_CURRENT_SOURCE_DIR}/RTOSWrapper/include
+        RTOSWrapper/include
+        prof
 
         )
 
 #Add FreeRTOS SystemvView headers if enabled
-if((${PROJECT_TARGET} STREQUAL "TARGET_RT1051") AND (${SYSTEM_VIEW_ENABLED}))
-    target_include_directories(${PROJECT_NAME}
-        PUBLIC
-            ${CMAKE_CURRENT_SOURCE_DIR}/systemviewPatch
-    )
+if ((${PROJECT_TARGET} STREQUAL "TARGET_RT1051") AND (${SYSTEM_VIEW_ENABLED}))
     target_link_libraries(${PROJECT_NAME} PUBLIC segger::systemview)
```

Comment:
Have you checked if systemview is working after changes?

Reply:
It didn't even compile before both for pure/bell. At least it is now compiling and linking properly ;)